com.mongodb.Compass.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "com.mongodb.Compass",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "21.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.node14"
+    ],
+    "command": "run.sh",
+    "separate-locales": false,
+    "tags": [],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--talk-name=org.freedesktop.secrets",
+        "--share=network"
+    ],
+    "modules": [
+        "shared-modules/libsecret/libsecret.json",
+        {
+            "name": "run",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 -t /app/bin/ run.sh"
+            ],
+            "build-options": {
+                "env": {
+                    "ZYPAK_SANDBOX_FILENAME": "chrome-sandbox",
+                    "ZYPAK_DEBUG": "1"
+                }
+            },
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "run.sh",
+                    "commands": [
+                        "zypak-wrapper '/app/main/MongoDB Compass'"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "compass",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp -r main /app/main",
+                "install -Dm 644 icon-64x64.png /app/share/icons/hicolor/64x64/apps/com.mongodb.Compass.png",
+                "install -Dm 644 icon-128x128.png /app/share/icons/hicolor/128x128/apps/com.mongodb.Compass.png",
+                "install -Dm 644 icon-256x256.png /app/share/icons/hicolor/256x256/apps/com.mongodb.Compass.png",
+                "install -Dm 644 mongodb-compass.desktop /app/share/applications/com.mongodb.Compass"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/mongodb-js/compass/releases/download/v1.32.2/mongodb-compass-1.32.2-linux-x64.tar.gz",
```

Comment:
since this is x86_64 only you need to instruct flathub to not build it

Reply:
aarch64 build is available as `.deb` package

https://github.com/mongodb-js/compass/releases/tag/v1.32.2

Reply:
I wish you were right here, but I don't see it. As for unpacking .deb, there are a lot of flatpak that do that.

You could build from source though.

Reply:
I'll give a look, thanks